AnkiDroid/src/main/java/com/ichi2/libanki/Card.java

```diff
@@ -662,4 +662,14 @@ public int hashCode() {
         // Map a long to an int. For API>=24 you would just do `Long.hashCode(this.getId())`
         return (int)(this.getId()^(this.getId()>>>32));
     }
+
+    public int getUserFlag() {
+        return mFlags & 0b111;
+    }
+
+    public void setUserFlag(int flag) {
+        assert(0<= flag);
```

Comment:
small whitespace nit

```suggestion
        assert(0 <= flag);
```

also, do we have an assert utility yet? because assert is otherwise useless on android